src/components/Table/Table.jsx

```diff
@@ -23,7 +23,7 @@ type Props = {|
   stickAt?: number,
   style?: StyleAttributes,
   data?: Data,
-  sort?: { key: string, order: string },
+  sort?: { key: ?string, order: ?string },
```

Comment:
This seems wrong. If you specify a sort order it should at least include these two values (or at least the key with a default of ASC order).

Reply:
Maybe... There was a flow issue in ss so this seemed to fix it